Gems/ROS2/Code/Source/RobotImporter/Utils/RobotImporterUtils.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include "AzCore/Component/ComponentBus.h"
+#include "AzCore/std/string/string.h"
+#include "RobotImporter/URDF/UrdfParser.h"
+#include <AzCore/IO/SystemFile.h>
+#include <AzCore/Math/Transform.h>
+#include <AzCore/std/containers/unordered_map.h>
+#include <AzCore/std/function/function_template.h>
+
+namespace ROS2
+{
+    namespace
+    {
+        const AZStd::function<bool(const AZStd::string&)> fileExistsCall = [](const AZStd::string& filename) -> bool
+        {
+            return AZ::IO::SystemFile::Exists(filename.c_str());
+        };
+    }
+
+    namespace Utils
+    {
+        bool IsWheelURDFHeuristics(const urdf::LinkConstSharedPtr& link);
+
+        //! The recursive function for the given link goes through URDF and finds world-to-entity transformation for us.
+        //! It traverses URDF from the given link to the root.
+        //! @param t - should be default (identity).
+        //! @returns root to entity transform
+        AZ::Transform GetWorldTransformURDF(const urdf::LinkSharedPtr& link, AZ::Transform t = AZ::Transform::Identity());
+
+        //! Retrieve all child links in urdf.
+        //! @param child links list of links in a query
```

Comment:
The @param value is wrong
```suggestion
        //! @param childLinks list of links in a query
```